wicker/testing/storage.py

```diff
@@ -83,18 +83,12 @@ def fetch_file(self, input_path: str, local_prefix: str, timeout_seconds: int =
         return target_path
 
     # Override.
```

Comment:
This `fetch_partial_file_s3` function below is not used anywhere and there are no other references to anything like this, so removing it.